src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -7449,13 +7449,13 @@ public unsafe void Append(ReadOnlySpan<byte> name, ReadOnlySpan<byte> value)
                 // Convert value to string first, because passing two spans causes 8 bytes stack zeroing in 
                 // this method with rep stosd, which is slower than necessary.
                 nameStr = name.GetHeaderName();
-                var valueStr = value.GetRequestHeaderString(nameStr, EncodingSelector);
+                var valueStr = value.GetRequestHeaderString(nameStr, EncodingSelector, checkForNewlineChars);
                 AppendUnknownHeaders(nameStr, valueStr);
             }
         }
 
         [MethodImpl(MethodImplOptions.AggressiveOptimization)]
-        public unsafe bool TryHPackAppend(int index, ReadOnlySpan<byte> value)
+        public unsafe bool TryHPackAppend(int index, ReadOnlySpan<byte> value, bool checkForNewlineChars = false)
```

Comment:
HPack is always HTTP/2 so checkForNewlineChars would always end up being true, no need to pass the argument on this code path.

Reply:
`var valueStr = value.GetRequestHeaderString(nameStr, EncodingSelector, checkForNewlineChars);` gets used by both `Append` and `TryHPackAppend` so I'll just define `checkForNewlineChars` as a var in `TryHPackAppend`